src/bundle/IbexaAutomatedTranslationBundle.php

```diff
@@ -21,6 +23,11 @@ public function getParent(): ?string
     public function build(ContainerBuilder $container): void
     {
         parent::build($container);
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__ . '/Resources/config'));
+
+        if ($container->hasExtension('ibexa_fieldtype_page')) {
```

Comment:
Prolly minor, as those two should be one package, but shouldn't it be `ibexa_page_builder`?

// and `page_builder_services.yaml`

Reply:
On the one hand, we have PageBuilderFieldEncoder and on the other hand, all dependencies are to `FieldTypePage`, that's why I gave that the required extension is `ibexa_fieldtype_page`